calicoctl/commands/node/status.go

```diff
@@ -129,21 +132,41 @@ func printBGPPeers(ipv string) {
 		os.Exit(1)
 	}
 
-	buf := make([]byte, 1024)
+	// BIRD socket read timeout.
+	timeOut := 2 * time.Second
+	bufReader := bufio.NewReader(c)
 
-	n, err := c.Read(buf[:])
-	if err != nil {
-		fmt.Printf("Error: %v\n", err)
+	birdOut := []string{}
+
+	for {
```

Comment:
1. Curious what value does scanner add vs using bufio bufferedReader? I've mostly seen scanner used for reading files and bufio reader for reading from socket.
2. I think we need to handle the error inside the loop, I looked at golang official example and they handle it inside for{} loop